### PR TITLE
Add min_encryption_version to the transit key response

### PR DESCRIPTION
**Pull request by meirish**

### CODE REVIEW

File: builtin/logical/transit/backend_test.go [hunk 1]
```diff
@@ -315,6 +317,12 @@ func testAccStepReadPolicy(t *testing.T, name string, expectNone, derived bool)
 			if d.Keys == nil {
 				return fmt.Errorf("bad: %#v", d)
 			}
+			if &d.MinDecryptionVersion == nil {
+				return fmt.Errorf("bad: %#v", d)
```
Comment (chrishoffman) [inline builtin/logical/transit/backend_test.go]: You shouldn't need to dereference here.

File: builtin/logical/transit/backend_test.go [hunk 2]
```diff
@@ -315,6 +317,12 @@ func testAccStepReadPolicy(t *testing.T, name string, expectNone, derived bool)
...
+			}
+			if &d.MinEncryptionVersion == nil {
```
Comment (chrishoffman) [inline builtin/logical/transit/backend_test.go]: Same here.

File: builtin/logical/transit/backend_test.go (hunk 1 above)
Comment (chrishoffman) [inline builtin/logical/transit/backend_test.go]: line 320, not 321 for this comment.

**Comment by meirish** [comment]
@chrishoffman updated - thanks for the 👀 !

**Pull request merged**